pages/incidents/index.js

```diff
@@ -1,41 +1,42 @@
+import React, { useContext } from "react";
 import styled from "styled-components";
+import IncidentForm from "../../components/incidentForm";
+import { useIncidents } from "../../context/listContext";
+
+function IncidentPage() {
+  const incidents = useIncidents().incidents;
+  const setIncidents = useIncidents().setIncidents;
+  console.log(incidents);
```

Comment:
Consider removing logs which you no longer need

Reply:
I was looking for that one since it always renders twice.